ACMILabs/interactive-label
# #30 31 32 hide cursor, highlight on touchstart, maybe disable zoom.

*Resolves issues #30, #31 and #32*

To test [outdated]:
- Hiding cursor with `startx -- -nocursor` has caused some displays to sleep. Make sure this doesn't happen!
- Highlighting on touchstart — do we need to unhighlight if the user drags the mouse away, without opening a label?
- Disabling zoom with a <meta name=viewport> tag, but this may only work on mobile? Needs to be tested on the actual device.

Update (3 april):
- I've been testing the touch functionality by connecting to my dev server from my phone. Get your local IP with ifconfig and navigate to http://[ip]:8081.
- The original <meta> zoom disable only works on mobile devices, so instead this uses a js touch event handler to prevent multitouch.
- We've disabled displays from sleeping with `xset s off -dpms`
